01 OperatorOverloading/OperatorOverloading.Host/Program.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OperatorOverloading.Model;
+namespace OperatorOverloading.Host
+{
+    class Program
+    {
+        public static void Main(string[] args)
+        {
+            Program programObject=new Program();
+            string amountTemporary;
+            string currencyTemporary;
+            try
+            {
+                Money moneyObject = new Money();
+
+                //Take input First Amount
+
+                Console.WriteLine("Enter Amount");
+                moneyObject.Amount =programObject.ValidateInputAmount();
```

Comment:
why do you need to make programObject to access the functions. You can directly  access them